api_v3/lib/types/KalturaGroupUser.php

```diff
@@ -91,4 +103,31 @@ public function getFilterDocs()
 	{
 		return array();	
 	}
+
+	public function validateForUpdate($sourceObject, $propertiesToSkip = array())
+	{
+		parent::validateForUpdate($sourceObject, $propertiesToSkip);
+		if(!kCurrentContext::$is_admin_session)
+		{
+			if($sourceObject->getUserRole() != $this->userRole)
```

Comment:
User with group-user update permission can change or update any group as before this feature.